benchopt/tests/__init__.py

```diff
@@ -61,12 +61,12 @@ def __exit__(self, exc_class, value, traceback):
 
         # Make sure to delete all the result that created by the run command.
         self.result_files = re.findall(
-            r'Saving result in: (.*\.csv)', self.output
+            r'Saving result in: (.*\.csv|.*\.parquet)', self.output
```

Comment:
We only save in `parquet` now, no? Not sure if we could have cases where both are produced/
```suggestion
            r'Saving result in: (.*\.parquet)', self.output
```